src/Uno.Extensions.Templates/content/unoapp-extensions/MyExtensionsApp.Shared/Styles/NavigationBar.xaml

```diff
@@ -1,26 +1,25 @@
-<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-                    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-                    xmlns:utu="using:Uno.Toolkit.UI">
+<ResourceDictionary
```

Comment:
might want to fix the formatting here

Reply:
It's an automated copy of the commerce app, we'll have to fix it there.